yoyodyne/models/pointer_generator.py

```diff
@@ -259,7 +259,7 @@ def decode_step(
                 last_h0.transpose(0, 1), features_enc, features_mask
             )
             # -> B x 1 x 4*hidden_size.
-            context = torch.cat([context, context], dim=2)
+            context = torch.cat([context, features_context], dim=2)
```

Comment:
Driveby, but would it make sense to call them `source_context` and `features_context`?

Reply:
True. Can do

Reply:
Okay, let me know when that's done and I'll merge.